platform/security/certificate/security-certificate-generator/src/main/java/org/codice/ddf/security/certificate/generator/CertificateCommand.java

```diff
@@ -163,11 +167,19 @@ public static String configureDemoCertWithDN(String[] dn, @Nullable String[] sub
     if (subjectAltNames != null) {
       csr.addSubjectAlternativeNames(subjectAltNames);
     }
-    RDN[] rdns = csr.getSubjectName().getRDNs(BCStyle.CN);
 
+    // Required for Chrome support (DDF-3104)
+    String commonName =
```

Comment:
❓ Would it not be easier to look back at the csr's x500 name and extract it from there? That way we leave all the parsing and logic to do so in one place.

Reply:
I don't think so. I looked at that class and I would have to do some backwards processing to arrive at the original CN.

Reply:
Then you will need to account for the fact that distinguished names are case insensitive